Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/person/ui/details/items/CwaUserCard.kt

```diff
@@ -38,11 +42,23 @@ class CwaUserCard(parent: ViewGroup) :
         }
     }
 
+    private fun formatBirthDate(dateOfBirthFormatted: String): String =
+        try {
+            LocalDate.parse(dateOfBirthFormatted, format).toDayFormat()
+        } catch (e: Exception) {
```

Comment:
This does not work for dates with only a year or a year and month as it add the missing values

Reply:
Fixed